src/app/modules/group/components/group-remove-button/group-remove-button.component.html

```diff
@@ -16,7 +16,7 @@
   <button
     pButton
     type="button"
-    class="p-button-rounded p-button-danger"
+    class="alg-button p-button-rounded p-button-danger"
```

Comment:
This is not the scope :thinking: Nice touch :+1: it would have been nice to have it in the PR notes to avoid the surprise :relaxed:

Reply:
Sure :)